src/openrct2-ui/windows/TrackDesignManage.cpp

```diff
@@ -22,6 +22,7 @@ static constexpr const int32_t WH = 44;
 static constexpr const int32_t WW = 250;
 static constexpr const int32_t WH_DELETE_PROMPT = 74;
 static constexpr const int32_t WW_DELETE_PROMPT = 250;
+static constexpr const int32_t TRACK_DESIGN_NAME_MAX_LENGTH = 127;
```

Comment:
For new local constants, please use TitleCase (per our [coding style](https://github.com/OpenRCT2/OpenRCT2/wiki/Coding-Style)). The other constants here (for the window) are still uppercase from when they were still preprocessor macros.

```suggestion
static constexpr const int32_t TrackDesignNameMaxLength = 127;
```